third_party/lint/repos.bzl

```diff
@@ -3,23 +3,28 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("//rules:repo.bzl", "http_archive_or_local")
 
-def lint_repos():
+def lint_repos(lowrisc_lint = None):
     # We have a 'vendored' copy of the google_verible_verilog_syntax_py repo
     native.local_repository(
         name = "google_verible_verilog_syntax_py",
         path = "hw/ip/prim/util/vendor/google_verible_verilog_syntax_py",
     )
 
+    # FIXME(cfrantz): Name collision with another package loading `com_github_bazelbuild_buildtools`?
+    # We seem to always be getting an _old_ version of buildtools that doesn't have the
+    # `buildify_test` rule.  Temporary workaround.
```

Comment:
Pinning to a specific version is probably fine in the long run: Lets us control when we take on merge skew risk.

Reply:
Its a best practice... thus spake the TL :)